lib/policies/jwt/jwt.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const passport = require('passport');
+const JWTStrategy = require('passport-jwt').Strategy;
+const extractors = require('./extractors');
+const services = require('../../services');
+
+module.exports = function (params) {
+  const secret = params.secretFile ? fs.readFileSync(params.secretFile) : params.secret;
+  const extractor = extractors[params.jwtExtractor](params.jwtExtractorField);
+
+  passport.use(new JWTStrategy({
+    secretOrKey: secret,
+    jwtFromRequest: extractor,
+    audience: params.audience
+  }, (jwtPayload, done) => {
+    if (!jwtPayload || !jwtPayload.iss) {
+      return done(null, false);
+    }
+
+    services.credential.getCredential(jwtPayload.iss, 'jwt')
```

Comment:
Per our discussion, we need a better way to uniquely identify a JWT credential, perhaps some combination of `iss`, `sub`, and `aud`.

Reply:
Done in 36205fdbabe6a3a846e908ef76f170d9419d9d46